e2e/install_test.go

```diff
@@ -998,56 +935,41 @@ func TestSingleNodeAirgapUpgradeCustomCIDR(t *testing.T) {
 
 	RequireEnvVars(t, []string{"SHORT_SHA"})
 
-	t.Logf("%s: downloading airgap files", time.Now().Format(time.RFC3339))
-	airgapInstallBundlePath := "/tmp/airgap-install-bundle.tar.gz"
-	airgapUpgradeBundlePath := "/tmp/airgap-upgrade-bundle.tar.gz"
+	tc := cmx.NewCluster(&cmx.ClusterInput{
```

Comment:
can we add comments to the `TestSingleNodeAirgapUpgradeCustomCIDR` and `TestSingleNodeAirgapUpgrade` entries in the test array within ci.yaml to say that they now use CMX?